pygrametl/tables.py

```diff
@@ -2123,6 +2128,11 @@ def __preparetempfile(self):
         self.__filename = self.__namedtempfile.name
         self.__ready = True
 
+    @property
+    def currentsize(self):
```

Comment:
Could `currentsize` be misunderstood as the size of the table in the database? Maybe `currentbulksize` could be used, but then the API differs between batch loading and bulk loading Dimensions/FactTables.

Reply:
Can be understood as the number of rows currently affected
https://www.python.org/dev/peps/pep-0249/#rowcount